src/api/useTerminologyServiceApi.ts

```diff
@@ -170,6 +170,50 @@ export class TerminologyServiceApi {
     }
     return null;
   }
+
+  getValueSetsForDRCs(cqmMeasure: CqmMeasure): ValueSet[] {
+    const drcValueSets = [];
+    const cqlCodes: CQL.CQLCode[] = this.getCqlCodesForDRCs(cqmMeasure);
+    if (cqlCodes) {
+      cqlCodes.forEach((cqlCode) => {
+        const valueSet = {
+          oid: `${cqlCode.code}`,
+          version: `${cqlCode.version}`,
+          concepts: [
+            {
+              code: `${cqlCode.code}`,
+              code_system_oid: `${cqlCode.oid}`,
+              code_system_name: `${cqlCode.system}`,
+              code_system_version: `${cqlCode.version}`,
+              display_name: `${cqlCode.display}`,
+            },
+          ],
+          display_name: `${cqlCode.display}`,
```

Comment:
looks like unnecessary template literals. you can removes `${}`

Reply:
This is done.